### PR TITLE
fix tester issue

### PR DESCRIPTION
**Pull request by markiyanshvets**

### CODE REVIEW

File: src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts [hunk 1]
```diff
@@ -75,18 +72,33 @@ export class EditSupplierComponent implements OnInit {
     this.closeDiv.nativeElement.click();
     const form = this.supplierForm.value;
 
+    let currentCurrency = null;
+    this.currencies.forEach(element => {
+      if(element.id== form.currency){
```
Comment (danylo-dudok) [inline src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts]: I think Array.filter will be more suitable for the case.

Comment (danylo-dudok) [reply src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts]: element.id === parseInt(form.currency)

**Review by danylo-dudok — CHANGES_REQUESTED** [review]

File: src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts (hunk 1 above)
Comment (markiyanshvets) [reply src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts]: okay

File: src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts [hunk 2]
```diff
@@ -75,18 +72,33 @@ export class EditSupplierComponent implements OnInit {
...
+    let currentCurrency = null;
+    this.currencies.forEach(element => {
+      if(element.id === parseInt(form.currency)){
+        currentCurrency = element;
+      }
+      
+    });
+
+    let currentCountry = null;
+    this.countries.forEach(element => {
```
Comment (TarasKvasnytsya) [inline src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts:84]: currentCountry = this.countries[this.countries.findIndex(element => element.id === parseInt(form.country))];

Comment (TarasKvasnytsya) [reply src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts:84]: I think it will be better then cycle and if operator

**Review by TarasKvasnytsya — APPROVED** [review]

File: src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts [hunk 3]
```diff
@@ -75,18 +72,33 @@ export class EditSupplierComponent implements OnInit {
     this.closeDiv.nativeElement.click();
     const form = this.supplierForm.value;
 
+    let currentCurrency = null;
+    this.currencies.forEach(element => {
```
Comment (danylo-dudok) [inline src/app/modules/shared/components/supplier/edit-supplier/edit-supplier.component.ts:76]: Array.filter

**Review by danylo-dudok — APPROVED** [review]

**Pull request merged**